packages/common/telemetry.ts

```diff
@@ -1,7 +1,12 @@
 import { post } from './fetchWrappers'
 
 export function handlePageTelemetry(API_URL: string, route: string, telemetryProps: any) {
-  const { page_url, search, language, viewport_height, viewport_width } = telemetryProps
+  const { page_url } = telemetryProps
+
+  // check if ph exists, if not use an empty object
+  const ph = telemetryProps?.ph || {}
```

Comment:
this looks wrong though - from ConstentToast we pass in `telemetryProps` which is derived from `useTelemetryProps()`, which returns this
https://github.com/supabase/supabase/blob/master/packages/common/hooks/useTelemetryProps.ts#L19

`ph` is not part of the return object

Reply:
The API endpoint is defined in `api.d.ts`. We should just use it here.